staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/deepcopy.go

```diff
@@ -268,5 +268,21 @@ func (in *JSONSchemaProps) DeepCopy() *JSONSchemaProps {
 		}
```

Comment:
it would make the PR much easier to review if this commit was split into logical pieces.